src/modules/endpoint/selectors/__tests__/isPeekingRoute.spec.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+import isPeekingRoute from '../isPeekingRoute'
+
+describe('isPeekingRoute', () => {
+  it('should return true if new city is equal to content city', () => {
```

Comment:
You're expecting false, so either change the description or the test. Also a test for the opposite is missing.